### ISSUE TITLE
Rendering icon, not match template configuration

### ISSUE THREAD

**Issue by afabri73** [skartknet/Iconic#37]
Hi,
the frontend result, not match template configuration.

### Configuration ###
![Iconic - SS2 - Settings - Iconic Configuration](https://github.com/skartknet/Iconic/assets/6714121/c032bceb-a3d4-4488-9b9f-be40259ca4df)

### Content Field ###
![Iconic - SS1 - Content - Icon](https://github.com/skartknet/Iconic/assets/6714121/a31cd7d1-9c43-469e-9cfa-5e6bfdae8345)

### Partial View ###
![Iconic - SS3 - Settings - Partial View](https://github.com/skartknet/Iconic/assets/6714121/f87b76d3-3984-4dc2-af8a-5f655d87faed)

### Frontend result - Icon ###
![Iconic - SS4 - Settings - Result](https://github.com/skartknet/Iconic/assets/6714121/05c40cc0-9c0e-477e-b690-1c195fa9b223)

### Frontend result - Inspector ###
![Iconic - SS4 - Settings - Result (1)](https://github.com/skartknet/Iconic/assets/6714121/993aef24-2c63-4e8b-9052-f913249f000f)

As you can see in the inspector, the rendered html not match the iconic template configuration
I would expect to see `<i class="fas fa-briefcase iconTop"></i> Lavorare in OPBG`, instead the code also includes again the TAG `<i>` which I didn't put inside the template:
`<!-- <i class="fas fa-<i class=&quot;fas fa-briefcase&quot;></i> iconTop"></i> Font Awesome fontawesome.com --> Lavorare in OPBG`

Can you help me to understand why isn't rendered only the icon as indicated in the template configuration?

Thank you for the support
Adriano

**Comment by skartknet** [comment]
Hey Adriano, take a lokk at [this doc](https://github.com/skartknet/Iconic/blob/master/Documentation/Usage/README.md#-displaying-the-icon)

You don't have to use `.Value()` You need to use `@Html.Raw(Model.Icon)` or `@Html.RenderIcon(Model.Icon)`

**Comment by skartknet** [comment]
I've release version 5.3.0. That hopefully fixes the problem.

**Comment by afabri73** [comment]
Hi,
thank you for the answer.
The problem is that I need to restart website to see the change.
I'll update the package to latest version that contain the fix.

A last thing...using Value, and after restarted website, I see correct value of the icon (in my case "fa-briefcase").
I don't need ro wrapped into Html.Raw or Html.RenderIcon
Value show the icon name (fa-briefcase)

Problem solved.
Adriano

**Comment by skartknet** [comment]
I have release 5.3.0 now.

**Comment by afabri73** [comment]
Thank you for the update.
Tomorrow I'll update it on my project.

Have a good day
Adriano

**Comment by nickwelsh1** [comment]
Thankyou @skartknet and @afabri73 ,

I had the same issue where anything I added to the template around {icon} would not come through to the frontend HTML.
Upgrading to 5.3.0 fixed this issue. 

A few other things I noticed while testing:

1. From a UX perspective why does "template" and "Enter a backoffice template" operate differently. Would it be good to allow users to specify <i class="las {icon} la-3x"></i> in the template (used on the frontend) and <i class="las {icon} la-1x"></i> in the backoffice template?

2. The "Rules source file" field description  says "if none selected, it will use the chosen CSS file" and yet it would not let save if this field is empty. This issue also exists in 5.3.0.
![image](https://github.com/skartknet/Iconic/assets/15595566/bf1b547e-6e6a-4464-b3a0-eea174a0f3fe)
 
 
3. Tested in Umbraco 11.4.0. Checking installed packages Iconic version lists "unkown". 
![image](https://github.com/skartknet/Iconic/assets/15595566/93fc5664-4805-4d82-b60a-abc4c0617516)
 
 
4. Is it possible to create our own Pre-Configurations?

Issues noticed while trying to get Line-awesome to work in my project.

Thank you for the support,
Nick

**Comment by skartknet** [comment]
Hi @nickwelsh1 ,

1. Not sure if I understand the question. 'Template' is used both in the frontend and backoffice to offer a preview of the icon. If you enter a value on 'Enter a backoffice template', this will override the value on Template and will be used on the backoffice instead. This could be useful for example if you want to rotate the icon in a loop but you don't want to do this on the preview.
2. You're right, I have opened a ticket to fix this. (#41)
3. Weird, I'll take a look. I've tested on v12 and it doesn't even show the Version label.
4. No, it's not possible. But it sounds like a good new feature. (#42)

**Comment by nickwelsh1** [comment]
Thankyou @skartknet,

I now notice my mistake regarding 1. 
I hadn't realised we specify the view CSHTML templates differently so that we can use 2x icons in some places and use 3x in others add classes. 


I wanted to swap from a Line-Awesome divided into sets (accessibility, alerts, animals, arrows, ... ), to a single Line-Awesome all icons, but when I tried I found content authors would have to reselect icons. Usync must remove references so it doesn't seem possible to do a swap. 

Have a good day,
Nick
